<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListControl</class>
 <widget class="QWidget" name="ListControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="imgLabel">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>85</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>85</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>90</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">
QPushButton{
  background-color: #F56C6C;
  border: 1px solid #F56C6C;
  border-radius: 4px;
  color: #fff;
  font-size: 13px;
  font-weight: 400;
  font-family: &quot;Microsoft YaHei&quot;, &quot;PingFang SC&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif;
  line-height: 24px;
  padding: 8px 15px;
}
QPushButton::hover {
  background-color: #f89898;
  border: 1px solid #f89898;
  color: #fff;
}
QPushButton::pressed {
  background-color: #c45656;
  border: 1px solid #c45656;
}
QPushButton::disabled{
  background-color: #fab6b6;
  border: 1px solid #fab6b6;
  color: #fff;
}
</string>
          </property>
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="indexLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
